src/ui/components/MenuButton/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { createUseStyles } from 'react-jss';
+import PropTypes from 'prop-types';
+import { useTheme } from '../../../contexts/ThemeContext';
+
+const useStyles = createUseStyles({
+  menu: {
+    backgroundColor: ({ theme }) => theme.container.background,
+    color: ({ theme }) => theme.translucent[70],
+    position: 'absolute',
+    right: ({ anchorElPosition }) =>
+      window.innerWidth - anchorElPosition?.right,
+    width: 200,
+    marginTop: 5,
+    borderRadius: 5,
+    border: ({ theme }) => `1px solid ${theme.container.outline}`,
+    cursor: 'pointer',
+    overflow: 'hidden',
+  },
+});
+
+function MenuButton({ children, element }) {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const anchorElPosition = anchorEl?.getBoundingClientRect();
```

Comment:
This returns the coordinates, width, height, etc of the element that was clicked. We can use these properties to figure out where to absolute position the `Menu` that opens.